ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource

```diff
@@ -46,6 +46,15 @@ Import Pipeline
     [Documentation]    Import a pipeline definition from DS Project details page.
     ...                It expects to receive the relative path starting from ods_ci/
     [Arguments]    ${name}    ${filepath}    ${project_title}    ${description}=${NONE}    ${press_cancel}=${FALSE}
+    # Begin - Workaround for https://github.com/opendatahub-io/data-science-pipelines-operator/issues/349
+    Capture Page Screenshot
+    Reload Page
```

Comment:
the issue says that the button is active before the pods are ready, right? Wouldn't wait for the server pods to be ready be a more stable workaround?